pom.xml

```diff
@@ -350,6 +350,19 @@
         </dependencies>
       </plugin>
 
+      <plugin>
+        <groupId>org.folio</groupId>
+        <artifactId>folio-module-descriptor-validator</artifactId>
+        <version>1.0.0</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>validate</goal>
```

Comment:
The usage of the folio-module-descriptor-validator works with this plugin configuration.